PubNubSwiftChatSDK/PubNubSwiftChatSDK.docc/Channel.md

```diff
@@ -0,0 +1,78 @@
+# ``PubNubSwiftChatSDK/Channel``
+
+## Topics
+
+### Receiving Updates
+
+- ``streamUpdates(callback:)``
+- ``streamUpdatesOn(channels:callback:)``
+- ``streamReadReceipts(callback:)``
+- ``streamMessageReports(callback:)``
+- ``streamPresence(callback:)``
+
+### Update and Delete a Channel
+
+- ``update(name:custom:description:status:type:completion:)``
+- ``delete(soft:completion:)``
+
+### Typing Indicator
+
+- ``startTyping(completion:)``
+- ``stopTyping(completion:)``
+- ``getTyping(callback:)``
```

Comment:
Is this intentional that in some places there is `callback` as parameter and in others `completion`?